app/views/templates/general_content.html.haml

```diff
@@ -1,7 +1,8 @@
-- breadcrumb :public_node, node
+%article#content{class: "content-main"}
+  - breadcrumb :public_node, node
```

Comment:
@tessereth All I've done is pull the article from the layout and into the partials. I haven't added any new HTML. 

The homepage is not inside an article, general content  pages are.